commercetools-java-client-core/src/main/java/io/sphere/sdk/client/TestDoubleSphereClientFactory.java

```diff
@@ -81,6 +86,11 @@ public void close() {
             public String toString() {
                 return "SphereClientObjectTestDouble";
             }
+
+            @Override
+            public SphereApiConfig getConfig() {
+                return null;
```

Comment:
I don't feel it right that in a class delivered by our library we have a method not signalled with `@Nullable` that returns a null. Of course we don't have anything meaningful to return because it's a fake client...
But checking the code I see that in one of the methods there is one SphereApiConfig with fake data. Maybe we can go on that line and provide a dummy config with a project key and API URL that leaves no doubt this is a fake config. Maybe @katmatt has a better idea?